RF24Mesh.cpp

```diff
@@ -288,6 +284,22 @@ bool ESBMesh<network_t, radio_t>::releaseAddress()
 
 /*****************************************************/
 
+#ifndef MESH_NO_MASTER
```

Comment:
I just realized this should be spelled `MESH_NOMASTER`. I'll submit a commit to master to fix this.